### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by shivamkm07**
This PR updates Readme to remove incorrect description that component files are created in airgap dapr init. It also updates workflow to use .txt file for Readme instead of markdown. 

Please reference the issue this PR will close: #14, #12

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -91,13 +92,18 @@ jobs:
           echo RUNTIME VERSION: ${{env.RUNTIME_VERSION}}
           echo GITHUB_EVENT_NAME: ${GITHUB_EVENT_NAME}
       
+      - name: Create Readme.txt file
+        shell: bash
+        run: |
+          cp ${{env.README_MD_FILE}} ${{env.README_TXT_FILE}}
```
Comment (mukundansundar) [inline .github/workflows/release.yaml:98]: Have a quick question regarding this, won't the formatting chars that we use for MD like `#`, backtick etc. be as it is in the README text file?

**Review by mukundansundar — APPROVED** [review]
lgtm. One comment.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (shivamkm07) [reply .github/workflows/release.yaml:98]: The characters would be there. But that should be okay as anyway it's a readme and the formatting characters do improve readabiliity. 
 
Having a separate readme text file in repo itself is something I don't like to do as then any change in Readme.md would need a change in txt file as well, causing redundancy and inconsistency between two

Comment (shivamkm07) [reply .github/workflows/release.yaml:98]: Or else we can keep .md file itself in the bundle if characters cause an issue?

**Comment by shivamkm07** [comment]
@dapr/maintainers-dapr Can we merge this?

**Review by artursouza — APPROVED** [review]

**Pull request merged**